src/participant/participant.ts

```diff
@@ -775,7 +778,12 @@ export default class ParticipantController {
   Interact with your MongoDB clusters and generate MongoDB-related code more efficiently with intelligent AI-powered feature, available today in the MongoDB extension.\n\n
   Please see our [FAQ](https://www.mongodb.com/docs/generative-ai-faq/) for more information.\n\n`)
       );
-      void this._storageController.update(
+
+      this._telemetryService.track(
+        TelemetryEventTypes.PARTICIPANT_WELCOME_SHOWN
+      );
+
+      await this._storageController.update(
```

Comment:
This shouldn't be a functional change as the `update` method is not actually async, but I'm not a huge fan of fire-and-forgetting promises.